config.yaml

```diff
@@ -136,21 +136,21 @@
     - overall_scores:
         files:
           - name: overall_scores
-            id: syn25575156.9
+            id: syn25575156.13
             format: table
         final_format: json
         custom_transformations: 1
         column_rename:
           genename: hgnc_gene_id
         provenance:
-          - syn25575156.9
+          - syn25575156.13
         agora_rename:
-          ensg: ENSG
-          hgnc_gene_id: GeneName
-          geneticsscore: GeneticsScore
-          literaturescore: LiteratureScore
-          overall: Logsdon
-          omicsscore: OmicsScore
+          ensg: ensembl_gene_id
+          hgnc_gene_id: hgnc_symbol
+          geneticsscore: genetics_score
+          literaturescore: literature_score
+          overall: target_risk_score
+          omicsscore: multi_omics_score
```

Comment:
Since you're already changing the names Agora expects, is there any reason why these can't go in the column_rename instead? It doesn't make sense that we're doing genename -> hgnc_gene_id -> hgnc_symbol either.

Reply:
1. No particular reason, other than it required fewer code changes. Looking through the config, we are doing essentially the same thing for the gene_info and network datasets too. I created https://sagebionetworks.jira.com/browse/AG-952 to follow up on moving as much of the aliasing as we can into column_rename.

2. I want to eventually remove hgnc fields from all files except gene_info (via https://sagebionetworks.jira.com/browse/AG-942) so we have only a single source of truth, but we'll need to ensure that the app isn't still using any of those first. For now, I was thinking we can just live with the current aliasing strategy.

Reply:
Ok, let's leave renaming rearrangement for that ticket and ignore it on this one.